internal/command/views/apply.go

```diff
@@ -121,10 +121,11 @@ func (v *ApplyHuman) Operation() Operation {
 }
 
 func (v *ApplyHuman) Hooks() []tofu.Hook {
-	return []tofu.Hook{
-		v.countHook,
-		NewUiHook(v.view),
+	hooks := []tofu.Hook{v.countHook}
+	if !v.view.concise {
+		hooks = append(hooks, NewUiHook(v.view))
```

Comment:
`suggestion`
To have this cleaner, I would suggest to pass the `concise` value to the `NewUiHook` function and return a `NoOpUiHook` instead.

Reply:
If we already had a NoOpUiHook, that would make a lot of sense.  As it is I think the smaller changeset is preferred :+1:

Reply:
After some more checks with @diofeher, there is such a thing: `tofu.NilHook`. With just a small change, we could use that instead.

Reply:
Exactly. Therefore, we were thinking on something like this:
```
type ConciseUiHook struct {
	tofu.NilHook
}

func NewUiHook(view *View, concise bool) tofu.Hook {
	if concise {
		return &ConciseUiHook{}
	}
	return &UiHook{ ... }
}
```
As @diofeher said, this can also be extended later if we need to.

Reply:
Let's give this approach a try, but keep NewUiHook the same.  We can add a new function next to it called something like `OptionalUiHook(view *View)`